examples/self_supervised/byol.py

```diff
@@ -0,0 +1,105 @@
+# flake8: noqa
+import argparse
+
+from common import add_arguments, ContrastiveModel, datasets
```

Comment:
could we use local naming?
like `from .common`?

Reply:
Ok, I will do that.

Reply:
👀